synapse/handlers/room.py

```diff
@@ -1721,6 +1760,46 @@ class ShutdownRoomResponse(TypedDict):
     new_room_id: Optional[str]
 
 
+@attr.s(slots=True, auto_attribs=True)
+class DeleteStatus:
+    """Object tracking the status of a delete room request
+
+    This class contains information on the progress of a delete room request, for
+    return by get_delete_status.
+    """
+
+    STATUS_PURGING = "purging"
+    STATUS_COMPLETE = "complete"
+    STATUS_FAILED = "failed"
+    STATUS_SHUTTING_DOWN = "shutting_down"
+    # Purge waiting to be launch at a specific time
+    STATUS_WAIT_PURGE = "wait_purge"
```

Comment:
Re-arranging these to be in order how we expect things to happen:

```suggestion
    STATUS_SHUTTING_DOWN = "shutting_down"
    # Purge waiting to be launch at a specific time
    STATUS_WAIT_PURGE = "wait_purge"
    STATUS_PURGING = "purging"
    STATUS_COMPLETE = "complete"
    STATUS_FAILED = "failed"
```